src/cbc_sdk/platform/alerts.py

```diff
@@ -997,6 +997,92 @@ def set_rows(self, rows):
         self._batch_size = rows
         return self
 
+    def set_time_range(self, *args, **kwargs):
```

Comment:
Should we remove `set_time_range` from https://github.com/carbonblack/carbon-black-cloud-sdk-python/blob/Alerts_v7_SDK_Changes/src/cbc_sdk/platform/legacy_alerts.py#L342 as these conflict on the Alert Query class